alifedata_phyloinformatics_convert/_impl/_try_alifestd_validate.py

```diff
@@ -0,0 +1,93 @@
+"""This file has been copied from hstrat (https://github.com/mmore500/hstrat/blob/47fe9048d6b327ca40f77424e3cf9392f7980689/hstrat/_auxiliary_lib/_alifestd_validate.py).
```

Comment:
Paste this link as a comment oon top of all copied files. Don’t put as a docstring

Reply:
175b4d9